scripts/init.sh

```diff
@@ -2,6 +2,18 @@
 
 set -e
 
+echo "Checkout polkadot-sdk Snowfork fork"
+pushd ..
+if [ ! -d "polkadot-sdk" ]; then
+  git clone https://github.com/Snowfork/polkadot-sdk.git
+fi
+pushd  polkadot-sdk
+git checkout snowbridge
+popd
+popd
+
+ln -sf ../polkadot-sdk polkadot-sdk
```

Comment:
Why do we need this if we are still using the submodule?

Reply:
Seems Clara removed it, but we may also need to update git modules accordingly.
https://github.com/Snowfork/snowbridge/blob/c1de2085dfeae6a6b1d155956aaa8bd06b879059/.gitmodules#L1-L3

Reply:
Good catch! 😄 https://github.com/Snowfork/snowbridge/pull/1131/commits/6b4d7457c80850dc0625c555997e28e5879226cc